config/functions

```diff
@@ -801,7 +801,15 @@ get_pkg_variable() {
 get_build_dir() {
   local _PKG_NAME="${1%:*}" _PKG_VERSION="$(get_pkg_version "$1")"
   if [ -n "$_PKG_NAME" -a -n "$_PKG_VERSION" ]; then
-    echo $BUILD/${_PKG_NAME}-${_PKG_VERSION}
+    echo $BUILD/build/${_PKG_NAME}-${_PKG_VERSION}
+  fi
+}
+
+# get package's install dir
+get_install_dir() {
+  local _PKG_NAME="${1%:*}" _PKG_VERSION="$(get_pkg_version "$1")"
+  if [ -n "$_PKG_NAME" -a -n "$_PKG_VERSION" ]; then
+    echo $BUILD/install_pkg/${_PKG_NAME}-${_PKG_VERSION}
   fi
 }
```

Comment:
This appears to be functionally equivalent to `$(get_pkg_variable PKG_INSTALL)`, and could be reduced to that so that `get_install_dir` becomes shorthand for `get_pkg_variable PKG_INSTALL`.